app/views/RoomActionsView/index.tsx

```diff
@@ -994,19 +956,70 @@ class RoomActionsView extends React.Component<IRoomActionsViewProps, IRoomAction
 		);
 	};
 
+	renderOmnichannelSection = () => {
+		const { room } = this.state;
+		const { rid, t } = room;
+		const { theme } = this.props;
+
+		if (t !== 'l' || this.isOmnichannelPreview) {
+			return null;
+		}
+
+		return (
+			<List.Section>
+				{this.omnichannelPermissions[0] ? (
```

Comment:
Will item number zero always be this permission?
Wouldn't it be better to have a helper making a map and returning the components according to the permission returned?

Because this check here only checks if it exists, if at some point the zero index is something else, it will keep returning this specific component.

In case the return is by the permission and not by the index, if it has the move-to-the-queue permission and not the chat-forward it will return the chat-forward permission, even if the permission is for something else.